test/kubemark/iks/config.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
This file should be placed at `cluster/kubemark/iks/config-default.sh` to follow convention.